Search results/embed viewer fix: centering issue

dominklenda opened the pull request:
Fix from the full-screen mode enhancements #95 

"search results page improvements: center article list div in left-panel, which will fix centering issue (same issue is currently affecting article page, see #64 ) and will fix the issue whereby the filter/sort buttons are being clipped on the right (see [screenshot](https://user-images.githubusercontent.com/4512335/76031693-36ad7000-5f06-11ea-83df-61bf88a806e1.png))"

I removed the column width constant and centered the article list div.

eplebel commented:
looks OK when zoomed out at 80%, but in default 100% zoom (for 1920 × 950 viewport), the article card width is too wide *and* the transprency filter popup is also being clipped.

  this is working correctly for the Recent and Author pages, so look there for clues on a fix.

![image](https://user-images.githubusercontent.com/4512335/91724815-da04d800-eb6b-11ea-8441-b43590c873ea.png)

Recent (where it's working)
![image](https://user-images.githubusercontent.com/4512335/91724875-ec7f1180-eb6b-11ea-8ece-66788c3d98a3.png)

dominklenda commented:
interesting, at my 100% it looks good, but still the left side of filter popup is aligned with the viewport border

![Screenshot from 2020-09-01 12-07-30](https://user-images.githubusercontent.com/43213967/91838661-1f99d180-ec4e-11ea-89aa-136dfd0cbc07.png)

dominklenda commented:
i moved the popup a bit to the right, so looks like this below now, but i'm not sure what to do with those differences between mine and your testing. i added Recent page for comparison 

![Screenshot_2020-09-01 Transparent credible evidence ](https://user-images.githubusercontent.com/43213967/91840525-1bbb7e80-ec51-11ea-9f09-1f0119ff6bf7.png)

![Screenshot_2020-09-01 Transparent credible evidence (1)](https://user-images.githubusercontent.com/43213967/91841454-afda1580-ec52-11ea-8b34-b667323b931c.png)

eplebel commented:
probably you have a larger viewport/screen resolution (because ya problem doesn't happen on my large monitor, only on my smaller laptop viewport)

dominklenda commented:
yes, you were right. i checked it for 1500 x 900 viewport (1499 is the threshold to show embedded content button) and the article list was too large. fixed it. now need to find a way to place filter popup in the left panel, because it still goes beyond the viewport (see the first picture). it appears that the same issue arises in the Recent page as well: the filter popup for small 1500 x 1900 viewport is too large (the second picture)

![Screenshot from 2020-09-01 14-44-36](https://user-images.githubusercontent.com/43213967/91853045-ecf9d400-ec61-11ea-87b3-dfb07906545d.png)

![Screenshot from 2020-09-01 14-44-38](https://user-images.githubusercontent.com/43213967/91853053-eff4c480-ec61-11ea-80b8-806ad9c5c04e.png)

eplebel commented:
>  you were right. i checked it for 1500 x 900 viewport (1499 is the threshold to show embedded content button) and the article list was too large. fixed it.

ok i hope your solution matches how it's working on the Recent and author pages

> now need to find a way to place filter popup in the left panel, because it still goes beyond the viewport (see the first picture). it appears that the same issue arises in the Recent page as well: the filter popup for small 1500 x 1900 viewport is too large

but that's a *different* problem (popup is being displayed underneath the embed viewer), and i remember Mick saying that this is a difficult problem.  so for now, just look to to the Recent and Author pages to see how the popup is specified differently to avoid being cut-off on the left (probably those positioning parameters)
